public_html/behavioralhealth/estacada/resources.html

```diff
@@ -130,345 +130,578 @@ <h1 class="blue no_margin some_margin" style="font-size: 40px;">Behavioral Healt
-                                    Outpatient Mental Health Treatment, Outpatient Substance Use Treatment, Incredible Years Parenting Groups, School-based Services.
+                                <h4 class="blue no_margin some_margin">DRUG & ALCOHOL HELPLINE</h4>
+                                <p> Phone: 800-923-4357 (24/7) OR Text “RecoveryNow” to 839863<br>
+                                    The 24/7 helpline is run by Lines for Life. It is free, confidential, and anonymous. Highly trained staff and volunteers provide immediate assistance, compassionate support, and resource referrals that can put you on a path to healing.
                                 </p>
                             </div>
                             <div class="pad_bottom">
-                                <h4 class="blue no_margin some_margin">LIFEWORKS NW - www.lifeworksnw.org</h4>
-                                <p>Gresham - 503-661-5455<br>
+                                <h4 class="blue no_margin some_margin">Q-CHAT</h4>
+                                <p> Website: <a target="_blank" href="https://www.qchatspace.org/">https://www.qchatspace.org/</a>
+                                </p>
+                            <div class="pad_bottom">
+                                <h4 class="blue no_margin some_margin">LIFEWORKS NW</h4>
+                                <p>Website: <a target="_blank" href="www.lifeworksnw.org">www.lifeworksnw.org</a><br>
+                                    Gresham - 503-661-5455<br>
                                     Rockwood - 503-645-9010<br>
                                     Gladstone - 503-594-1772<br>
                                     Milwaukie - 503-496-3201<br>
-                                    Child and family mental health counseling; Adult mental health counseling; Youth alcohol and drug addiction treatment;
+                                    Child and family mental health counseling; Adult mental health counseling; Youth alcohol and drug addiction treatment
                                 </p>
                             </div>
                             <div class="pad_bottom">
-                                <h4 class="blue no_margin some_margin">CLACKAMAS COUNTY BEHAVIORAL HEALTH - 503-655-8401</h4>
-                                <p>2051 Kaen Road, Oregon City, OR 97045<br>
+                                <h4 class="blue no_margin some_margin">SANDY BEHAVIORAL HEALTH CENTER</h4>
+                                <p> Phone: 503-722-6950<br>
+                                    38872 Proctor Blvd., Sandy, OR 97055<br>
+                                    Website: <a href="https://www.clackamas.us/healthcenters/sandy.html" target="_blank">www.clackamas.us/healthcenters/sandy.html</a><br>
                                     Serving children, youth, families and adults who are experiencing moderate to severe mental health difficulties, and have Oregon Health Plan or don’t have insurance.
```

Comment:
didn't get the new sentence. should read:
The Sandy Behavioral Health Center offers a variety of outpatient counseling and treatment services for both mental health and drug and alcohol issues.